openslides/mediafiles/views.py

```diff
@@ -15,6 +15,13 @@ class MediafileViewSet(ModelViewSet):
     access_permissions = MediafileAccessPermissions()
     queryset = Mediafile.objects.all()
 
+    def get_queryset(self):
```

Comment:
Forgot about that bit, added it. However, I'm getting errors when logged in, which aren't caused by my changes (checking out upstream/master causes the same issues). Are these known issues?

What I did was rebase on upstream/master, and run `./manage.py migrate`.

![capture](https://cloud.githubusercontent.com/assets/963290/18061426/f55ff2d0-6e22-11e6-8988-43bc9c074663.PNG)